core/src/client/mod.rs

```diff
@@ -1,7 +1,10 @@
+use std::str::FromStr;
```

Comment:
do you use this?

Reply:
Yes I need it to use `from_str` in `FieldElement::from_str(&address.to_string())` line 820, or maybe there's a better way to convert a string to a FieldElement ?

Reply:
If you want, you can use

`let a = felt!("0x1939391919313");`

Reply:
`felt!("")` expects a string literal, is it possible to convert a variable of type `&str` or `String` into a string literal ? I couldn't find a way to do it.

Reply:
to me, string literals are &str